app/src/app/app.component.ts

```diff
@@ -59,28 +59,36 @@ export class AppComponent implements AfterViewInit {
 
     // User log in
     public login(): void {
-        let loginSubscription = this.authenticationService.login().subscribe({
-            next: (res) => { this.checkStatus(); },
+        const loginSubscription = this.authenticationService.login().subscribe({
+            next: () => { this.checkStatus(); },
             complete: () => { loginSubscription.unsubscribe(); }
         });
     }
 
     // User log out
     public logout(): void {
-        let logoutSubscription = this.authenticationService.logout().subscribe({
+        const logoutSubscription = this.authenticationService.logout().subscribe({
             complete: () => {
                 this.router.navigate(['']);
-                logoutSubscription.unsubscribe(); 
+                logoutSubscription.unsubscribe();
             }
         });
     }
 
+    // User registration
+    public register(): void {
+        const registerSubscription = this.authenticationService.register().subscribe({
+            next: (res) => { this.checkStatus(); },
```

Comment:
Once a user registers, we should have an indication of whether or not the registration was successful. Currently, if a user registers successfully, a warning alert displays that their account is `pending` since _all_ registered accounts are assigned a `pending` status. The warning alert may cause concern that registration failed.